hstream.proto

```diff
@@ -346,6 +354,16 @@ message ReadShardResponse { repeated ReceivedRecord receivedRecords = 1; }
 message ListShardReadersRequest {}
 message ListShardReadersResponse { repeated string readerId = 1; }
 
+message ReadShardStreamRequest {
+  string readerId = 1;
+  uint64 shardId = 2;
+  ShardOffset shardOffset = 3;
+  uint32 timeout = 4;
+  uint32 maxRecords = 5;
```

Comment:
`maxRecords` is a parameter required by the underlying storage read interface, indicating that a read call waits for a maximum of `maxRecords` data before returning. Note it does not indicate that the current shardReader will only read a maximum of `maxRecords` data.

Reply:
How about renaming `maxRecords` to `maxBatched`?

Reply:
maybe `maxBatchedPerRead` or `maxBatchedWaited`?